conf/distro/include/ostro-xt-security-flags.inc

```diff
@@ -0,0 +1,3 @@
+SECURITY_CFLAGS_pn-crosswalk = "${SECURITY_NO_PIE_CFLAGS} ${SECURITY_PIC_CFLAGS}"
+SECURITY_LDFLAGS_remove_pn-crosswalk = "-fstack-protector-strong"
+SECURITY_LDFLAGS_append_pn-crosswalk = " -fstack-protector-all"
```

Comment:
`-fstack-protector-all` will introduce performance penalty, so this is not the best workaround.